flexmeasures/cli/monitor.py

```diff
@@ -69,18 +70,48 @@ def send_monitoring_alert(
     default="",
     help="Add this message to the monitoring alert (if one is sent).",
 )
-def monitor_tasks(task, custom_message):
+@click.pass_context
+def monitor_task(ctx, task, custom_message):
     """
+    DEPRECATED, use `latest-run`.
```

Comment:
My opinion is that such a hard deprecation is worthy of a release warning in the changelog.